atlas/templates/Jenkinsfile.gotmpl

```diff
@@ -41,5 +41,11 @@ pipeline {
       }
     }
   }
+  post {
+      always {
+          sh "cd $DIRECTORY && make clean || true"
+          cleanWs()
```

Comment:
What is this `cleanWs()` and why do we need this?

Reply:
this will cleanup the workspace
https://jenkins.io/doc/pipeline/steps/ws-cleanup/#cleanws-delete-workspace-when-build-is-done